interdiff_test.go

```diff
@@ -16,7 +18,23 @@ var interDiffFileTests = []struct {
 	{"test_examples/result_2.txt", "test_examples/diff_2_a.txt", "test_examples/diff_2_b.txt"},
 }
 
+// Reference: https://www.programming-books.io/essential/go/normalize-newlines-1d3abcf6f17c4186bb9617fa14074e48
+// NormalizeNewlines normalizes \r\n (windows) and \r (mac)
+// into \n (unix)
+func NormalizeNewlines(d []byte) []byte {
+	// replace CR LF \r\n (windows) with LF \n (unix)
+	d = bytes.Replace(d, []byte{13, 10}, []byte{10}, -1)
+	// replace CF \r (mac) with LF \n (unix)
+	d = bytes.Replace(d, []byte{13}, []byte{10}, -1)
+	return d
+}
+
+func Init() {
```

Comment:
```suggestion
func init() {
```

If you use lower case `init()` it'll be called automatically: https://golang.org/ref/spec#Package_initialization